### PR TITLE
Talk background to docs

### PR DESCRIPTION
**Pull request by schelvan**

### CODE REVIEW

File: talk/background.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+##############################
+Background Story of Sesam Talk
+##############################
+
+In 2022, we set out to explore the potential of Sesam Data Hub to establish a “plug and play” data synchronization service for SaaS business applications. This service would enable users and system owners to seamlessly onboard their systems, facilitating communication between them. In today’s competitive business landscape, organizations often favor best-of-breed solutions over comprehensive legacy business suites. However, seamless data flow across systems typically requires integration expertise and substantial project investments. Our objective was to identify ways to reduce these barriers, both technically and cost-effectively, for organizations seeking improved data quality, consistency, and reduced manual data entry. While the service would not encompass every system property, it would focus on Master Data and other core data types. This concept was christened Sesam Talk.
```
Comment (erik-leven) [inline talk/background.rst:5]: ...to seamlessly onboard their systems `and facilitate `communication between the...

File: talk/background.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+Sesam Talk Connectors
+=====================
+To facilitate seamless communication between systems, Sesam Talk necessitated a standardized connection method. Consequently, we developed specialized connectors for each system, comprising two primary components: the technical connector and the data mapping component.
+
+The technical connector primarily focused on the SaaS systems’ API endpoints, webhooks, authentication mechanisms, and other relevant functionalities. 
+
+The data mapping component, on the other hand, was responsible for mapping the properties of each SaaS system to the Sesam Talk data model. This process also included providing semantic context and descriptions for the properties, thereby enabling differentiation between entities such as human objects from SaaS systems, categorizing them as customers, employees, or both, and distinguishing between suppliers and customers based on organizational data collected from systems. 
+
```
Comment (erik-leven) [inline talk/background.rst:22]: The connectors had no information about the model. This happened outside the connectors in the enrich phase

Comment (erik-leven) [reply talk/background.rst:22]: The "mapping component" should probably be a section about the tooling

File: talk/background.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Sesam Talk in practice
+======================
+Upon onboarding a tenant’s SaaS system to Sesam Talk, the connector automatically generates Sesam Data Hub pipes and configuration for that tenant based on the mapping, semantic descriptions, and internal tooling. The data is collected, enriched, and placed into Globals as new objects or merged with existing objects that have been collected from other onboarded systems.
```
Comment (erik-leven) [inline talk/background.rst:26]: semantic descriptions? Maybe talk about the model here instead?

**Review by BaardBouvet — APPROVED** [review]

**Pull request merged**